examples/viewport.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>JS Bin</title>
+  <style>
+iframe {
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+}
+  </style>
+</head>
+<body>
+
+  <iframe src="everything.amp.html#paddingTop=54&webview=1"
+          frameborder="0" width="100%" height="100%"
+          scrolling="no">
+  </iframe>
+
+</body>
+</html>
```

Comment:
Added by mistake? If not, I'd update the filename to "everything.iframed.html" and `<title>` to "Test everything.amp.html as iframed webview.".